packages/console/src/pages/Applications/index.module.scss

```diff
@@ -3,17 +3,21 @@
 .headline {
   display: flex;
   justify-content: space-between;
+  padding: _.unit(6) _.unit(6) 0;
 }
 
 .table {
-  margin-top: _.unit(6);
-  width: 100%;
+  margin: _.unit(4) _.unit(6) _.unit(6);
 
-  tbody tr.clickable {
-    cursor: pointer;
+  tbody {
+    max-height: calc(100vh - _.unit(64));
```

Comment:
not cool for this. what if the table is inside a modal?

Reply:
Sorry the PR was merged before your comment. This tbody max-height thing is the only flaw in current solution. Will think about this and create another PR if there is any update.